core/system/src/mpu/vmpu.c

```diff
@@ -165,12 +165,76 @@ static void vmpu_sanity_check_box_namespace(int box_id, const char *const box_na
     } while (box_namespace[length]);
 }
 
+static void vmpu_box_index_init(uint8_t box_id, const uint32_t index_size, const UvisorBoxConfig *const config)
+{
+    void *box_bss;
```

Comment:
Place spaces around the `*`. We decided that was our style after you first wrote this commit.

Use this to predict the future next time. 🎱